### ISSUE TITLE
Heroku deployment broken after extensions migration

### ISSUE THREAD

**Issue by tomodachi94** [tomodachi94/dash2#11]
Heroku deployment doesn't appear to work.

The behavior of the `os.getcwd` function appears to return `/app/`, whereas on a local machine it is `/home/me/dashproj/dash`. Note that the `dash` folder at the end, which contains the project's entire source, has been excluded from `os.getcwd` on Heroku.

### Heroku
```log
app[worker.1]: Thank you for using lightbulb!
app[worker.1]: Traceback (most recent call last):
app[worker.1]: File "/app/dash/main.py", line 20, in <module>
app[worker.1]:     bot.load_extensions_from(DASH_EXT_PATH)
app[worker.1]:   File "/app/.heroku/python/lib/python3.9/site-packages/lightbulb/app.py", line 586, in load_extensions_from
app[worker.1]:     raise FileNotFoundError(f"'{path}' is not an existing directory")
app[worker.1]: FileNotFoundError: 'ext' is not an existing directory
heroku[worker.1]: Process exited with status 1
heroku[worker.1]: State changed from up to crashed
```
### Debugging
```
$ heroku run python
Python 3.9.7 (default, Aug 30 2021, 22:04:11) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.path.abspath(os.getcwd())
'/app'
```

## Workarounds
- [Detecting if the environment is Heroku](https://stackoverflow.com/questions/9383450/how-can-i-detect-herokus-environment) and adapting to the platform seems like a rather hacky solution.
- Hardcoding paths and recursively checking each one for a folder named `ext`. Again, rather hacky.
- Cross fingers and hope this is a bad dream :crossed_fingers:

**Comment by tomodachi94** [comment]
Turns out, this is the normal behavior of `os.getcwd`. It does not always return the folder in which the `.py` file resides. `__file__` contains the exact path and file, I'll use this with some list magic to make it work.

**Comment by tomodachi94** [comment]
Fixed with commit 62c26d39a03fa7f84f79078c513c11d84947145f.
